lib/virtual-fs/src/lib.rs

```diff
@@ -336,6 +336,9 @@ pub trait VirtualFile:
     /// the time at which the file was created in nanoseconds as a UNIX timestamp
     fn created_time(&self) -> u64;
 
+    /// sets accessed and modified time
+    fn set_times(&mut self, _atime: Option<u64>, _mtime: Option<u64>) {}
```

Comment:
Note: try to implement this for all the other implementations of VirtualFS.

We also need it for the `Dir`, and `Filesystem`